CSharp/BatchExplorer/ViewModels/ResizePoolViewModel.cs

```diff
@@ -143,6 +231,62 @@ public CommandBase ResizePool
             {
                 Messenger.Default.Send(new GenericDialogMessage(e.ToString()));
             }
+        }
+
+        private async Task EnableAutoScaleAsync()
+        {
+            try
+            {
+                if (this.IsAutoScale && IsInputValidForAutoScaling())
+                {
+                    Task asyncTask = this.batchService.EnableAutoScaleAsync(this.PoolId, this.AutoScaleFormula);
+
+                    AsyncOperationTracker.Instance.AddTrackedOperation(new AsyncOperationModel(
+                        asyncTask,
+                        new PoolOperation(PoolOperation.EnableAutoScale, this.poolId)));
+
+                    await asyncTask;
+                    Messenger.Default.Send(new CloseGenericPopup());
+                }
+            }
+            catch (Exception e)
+            {
+                Messenger.Default.Send(new GenericDialogMessage(e.ToString()));
+            }
+        }
+
+        private async Task EvaluateAsync()
+        {

```

Comment:
Putting Evaluate only in the resize UI seems a bit odd since you don't have to be resizing to evaluate your AutoScale formula -- what do you think about making a new option on the Pool right click context menu called "evaluate autoscale formula" instead?